test/Microsoft.Extensions.DataIngestion.Tests/Chunkers/SemanticChunkerTests.cs

```diff
@@ -1,39 +1,36 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Azure;
+using Azure.AI.OpenAI;
 using Microsoft.Extensions.AI;
 using Microsoft.Extensions.DataIngestion.Chunkers;
-using OpenAI;
+using Microsoft.ML.Tokenizers;
 using OpenAI.Embeddings;
 using System;
-using System.ClientModel;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Xunit;
 
-namespace Microsoft.Extensions.DataIngestion.Tests.Chunkers
+namespace Microsoft.Extensions.DataIngestion.Chunkers.Tests
 {
     public class SemanticChunkerTests : DocumentChunkerTests
     {
-        protected override IDocumentChunker CreateDocumentChunker()
+        protected override IDocumentChunker CreateDocumentChunker(int maxTokensPerChunk = 2_000, int overlapTokens = 500)
         {
             EmbeddingClient embeddingClient = CreateEmbeddingClient();
-            return new SemanticChunker(embeddingClient.AsIEmbeddingGenerator());
+            Tokenizer tokenizer = TiktokenTokenizer.CreateForModel("gpt-4o");
+            return new SemanticChunker(embeddingClient.AsIEmbeddingGenerator(), tokenizer, maxTokensPerChunk, overlapTokens);
         }
 
         private EmbeddingClient CreateEmbeddingClient()
         {
             string endpoint = Environment.GetEnvironmentVariable("AZURE_OPENAI_ENDPOINT")!;
             string key = Environment.GetEnvironmentVariable("AZURE_OPENAI_API_KEY")!;
 
-            return new EmbeddingClient(
-                "text-embedding-3-small",
-                credential: new ApiKeyCredential(key),
-                options: new OpenAIClientOptions()
-                {
-                    Endpoint = new Uri(endpoint)
-                }
-            );
+            AzureOpenAIClient openAIClient = new(new Uri(endpoint), new AzureKeyCredential(key));
```

Comment:
the code that was used here should have just worked, but for some reason I kept getting "Recource not found" errors. I suspect @KrystofS you were using slightly different endpoint address.